app/views/archives/index.html.haml

```diff
@@ -1,21 +1,19 @@
 %h1 Archives
-
-- @archives.each do |account, archives|
-  %h2= "#{account.name} (#{account.number})"
-  .row
-    %table.table.table-hover
-      %thead
-        %tr
-          %th #
-          %th Name
-          %th Month
-          %th Number left to explain
-          %th Download
-      %tbody
-        - archives.each do |archive|
+.row
+  %table.table.table-hover
+    %thead
+      %tr
+        %th #
+        %th Name
+        %th Month
+        %th Number left to explain
+        %th Download
+    %tbody
+      - if @archives
```

Comment:
`@archives` can never be nil here, can it?
Anyway, that validation should be done in the controller, not in the view.